docusaurus/docs/create-a-plugin/develop-a-plugin/best-practices.md

```diff
@@ -0,0 +1,64 @@
+---
+id: best-practices
+title: Best practices
+description: An index of best practices for plugins.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - best practices
+sidebar_position: 5
+---
+
+This document summarizes best practices for Grafana plugins. This live document is updated as new best practices are identified.
+
+## General
+
+- Choose the right plugin type for your use case: panel, data-source or app. See [Plugin types](/introduction/plugin-types-usage) for more information.
```

Comment:
Maybe too basic for a "best practice"? Or is this about reminding users to understand the fundamentals - if so we could reword.

Reply:
yeah, I wanted to start from the basics, agree that it's not that useful though, should I remove it?

Reply:
We could have something like "Never forget the fundamentals, be sure to read up on [introductory](/introduction) concepts for plugin development. 

(Haven't checked link syntax)

Reply:
that's a bit too vague, I can just remove it